test/e2e-olm/utils.go

```diff
@@ -46,22 +46,22 @@ func registerSchemes(t *testing.T) {
 	}
 }
 
-func createElasticsearchCR(t *testing.T, f *framework.Framework, ctx *framework.TestCtx, uuid string) (*elasticsearch.Elasticsearch, error) {
+func createElasticsearchCR(t *testing.T, f *framework.Framework, ctx *framework.TestCtx, uuid string, replicas int) (*elasticsearch.Elasticsearch, error) {
 	namespace, err := ctx.GetNamespace()
 	if err != nil {
 		return nil, fmt.Errorf("Could not get namespace: %v", err)
 	}
 
-	cpuValue := resource.MustParse("100m")
-	memValue := resource.MustParse("2Gi")
+	cpuValue := resource.MustParse("200m")
+	memValue := resource.MustParse("1Gi")
 
 	esDataNode := elasticsearch.ElasticsearchNode{
 		Roles: []elasticsearch.ElasticsearchNodeRole{
 			elasticsearch.ElasticsearchRoleClient,
 			elasticsearch.ElasticsearchRoleData,
 			elasticsearch.ElasticsearchRoleMaster,
 		},
-		NodeCount: int32(1),
```

Comment:
the reason this is `1` here is because that is how CLO generates the ES CR

Reply:
Taking into account that this is only a creator function. I left the decision to the calling side to use the default or not, e.g. for a multiple nodes cluster I want to start with two nodes already.